_docs/tasks/traffic-management-v1alpha3/ingress.md

```diff
@@ -47,15 +48,19 @@ This task describes how to configure Istio to expose a service outside of the se
 
 * Generate a certificate and key that will be used to demonstrate a TLS-secured gateway
 
-   A private key and certificate can be created for testing using [OpenSSL](https://www.openssl.org/).
```

Comment:
Please undo all the spacing/indent changes in this PR. This is unnecessary and is causing the presubmit test to fail.

Reply:
Holy hell, VS Code or something. Apologies.